moveit_core/kinematic_constraints/src/utils.cpp

```diff
@@ -273,3 +274,298 @@ moveit_msgs::Constraints kinematic_constraints::constructGoalConstraints(const s
 
   return goal;
 }
+
+static double parseDouble(XmlRpc::XmlRpcValue& v)
```

Comment:
Please provide some const specifiers to the arguments.

Reply:
Please neglect my comment. Obviously XmlRpc doesn't correctly declare constness of cast operations.

Reply:
Yes, the `XmlRpc` API is rather badly designed. This shows in a number of places in the request.
`int` for array indices, no iterators for arrays, missing `const` specifies in API functions, ...